spec/support/helpers/flickr_api_stub_helper.rb

```diff
@@ -0,0 +1,105 @@
+          'title': 'DSC_0093', 'ispublic': 1, 'isfriend': 0, 'isfamily': 0 },
+        { 'id': '24304500903', 'owner': '131891481@N05', 'secret': '11b9ef705d', 'server': '1638', 'farm': 2,
+          'title': 'DSC_0193', 'ispublic': 1, 'isfriend': 0, 'isfamily': 0 },
+        { 'id': '24838013681', 'owner': '131891481@N05', 'secret': 'efa5682a8a', 'server': '1640', 'farm': 2,
+          'title': 'DSC_0182', 'ispublic': 1, 'isfriend': 0, 'isfamily': 0 },
+        { 'id': '24813391572', 'owner': '131891481@N05', 'secret': '288f2d6441', 'server': '1533', 'farm': 2,
+          'title': 'DSC_0147', 'ispublic': 1, 'isfriend': 0, 'isfamily': 0 }
+      ] }, 'stat': 'ok' }
+
+  SEARCHED_PHOTO_RESULT = [
+    'https://farm2.staticflickr.com/1667/24503450029_3a043b85b1_q.jpg',
+    'https://farm2.staticflickr.com/1557/24753089122_7103b52410_q.jpg',
+    'https://farm2.staticflickr.com/1485/24574860910_498f9b0e25_q.jpg',
+    'https://farm2.staticflickr.com/1622/24844673716_1d598fea4b_q.jpg',
+    'https://farm2.staticflickr.com/1566/24777399031_5e69c152c0_q.jpg',
+    'https://farm2.staticflickr.com/1710/24503208499_bdfb1e2583_q.jpg',
+    'https://farm2.staticflickr.com/1650/24575342780_e66b987bc6_q.jpg',
+    'https://farm2.staticflickr.com/1672/24870790205_695e2dee96_q.jpg',
+    'https://farm2.staticflickr.com/1538/24844531776_117088d677_q.jpg',
+    'https://farm2.staticflickr.com/1688/24777365301_5fe568e0c1_q.jpg'
+  ]
+
+  RESEND_PHOTO_RESULT = [
+    'https://farm2.staticflickr.com/1690/24302286974_e0c00e989f_q.jpg',
+    'https://farm2.staticflickr.com/1593/24302287364_a071fd6a88_q.jpg',
+    'https://farm2.staticflickr.com/1572/24302287374_654f2758c9_q.jpg',
+    'https://farm2.staticflickr.com/1699/24302287604_b6753d19f9_q.jpg',
+    'https://farm2.staticflickr.com/1502/24302287934_b50fa68974_q.jpg',
+    'https://farm2.staticflickr.com/1718/24306046133_cd20720b0c_q.jpg',
+    'https://farm2.staticflickr.com/1605/24306046523_5d004949ce_q.jpg',
+    'https://farm2.staticflickr.com/1477/24814934032_dbacd8038c_q.jpg',
+    'https://farm2.staticflickr.com/1708/24814934672_e4e19300e7_q.jpg',
+    'https://farm2.staticflickr.com/1480/24814934682_89d17c58f1_q.jpg'
+  ]
+
+  def flickr_responce_stub
+    recent_response = RECENT_RESPONCE.to_json
+
+    search_response = SEARCH_RESPONCE.to_json
+
+    @searched_photo_result = SEARCHED_PHOTO_RESULT
+
+    @resent_photo_result = RESEND_PHOTO_RESULT
+
+    stub_request(:get, "https://api.flickr.com/services/rest?api_key="\
+      "#{FlickRaw.api_key}&extras=&format=json&"\
+      "method=flickr.photos.getRecent&nojsoncallback=1&per_page=10").
+      to_return(status: 200, body: "#{recent_response}", headers: {})
+
+    stub_request(:get, "https://api.flickr.com/services/rest?api_key="\
+      "#{FlickRaw.api_key}&format=json&"\
+      "method=flickr.photos.search&nojsoncallback=1&per_page=10&text=coffee").
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.